### PR TITLE
clean python code with standards

### PR DESCRIPTION
**Pull request by neodark**
I have added some standards to your code repository.

Hope you will like it and merge it.
Best regards!

### CODE REVIEW

File: .isort [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[settings]
+multi_line_output=3
+include_trailing_comma=true
+force_grid_wrap=0
+use_parentheses=true
+ensure_newline_before_comments=true
+line_length=88
+force_single_line=true
+order_by_type=true
+lines_between_types=1
```
Comment (itsbadr) [inline .isort:10]: this file doesn't add any value to the code unless you're using a specific formatter

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      - id: check-ast
+      - id: check-case-conflict
+      - id: trailing-whitespace
+      - id: end-of-file-fixer
+        exclude: version.txt
+      - id: debug-statements
+      - id: check-added-large-files
+      - id: check-docstring-first
+      - id: check-yaml
+        exclude: conda/meta.yaml
```
Comment (itsbadr) [inline .pre-commit-config.yaml:25]: exclude this file

**Comment by itsbadr** [comment]
sorry but this PR adds breaking changes like adding a comma at the end of args in specific __init__ functions. this is clearly made using an autoforamtter.

**Comment by neodark** [comment]
But this is the purpose of using black as a code formatter. I am not sure I understand your comment (although I had to improve  the comments at https://github.com/azure-badr/maaldar/blob/main/color-picker/main.py)

The contribution here was to improve your commits with `pre-commit-config.yaml` and `.isort` so you can have a pass of black and isort on your files by using it.

Not sure this PR was properly understood as it adds enhancement to your code. Anyway as you wish that's your repository of course.

**Pull request closed without merging**